neuromation/api/storage.py

```diff
@@ -713,3 +743,22 @@ def leave(self, data: StorageProgressLeaveDir) -> None:
 
     def fail(self, data: StorageProgressFail) -> None:
         pass
+
+
+def retries(msg: str) -> Iterator[ContextManager[None]]:
+    sleeptime = 0.0
+    for r in range(ATTEMPTS)[::-1]:
```

Comment:
Why is reversed iteration here?
I believe that the thing may be implemented simpler:
```
for r in range(ATTEMPTS):
    ...
    yield retry()
else:
    raise last_error
```

Reply:
Good idea!